internals/secrethub/generate_test.go

```diff
@@ -20,111 +21,141 @@ func newIntValue(v int) intValue {
 
 func TestGenerateSecretCommand_run(t *testing.T) {
 	testErr := errio.Namespace("test").Code("test").Error("test error")
+	testData := []byte("random generated secret")
+	testPath := "namespace/repo/secret"
 
 	cases := map[string]struct {
 		cmd               GenerateSecretCommand
 		writeFunc         func(path string, data []byte) (*api.SecretVersion, error)
 		clientCreationErr error
```

Comment:
This should be named `newClientErr` so it is consistent with the naming in other tests.

Reply:
Fixed